### PR TITLE
Created rough outline for a responsive design

### PR DESCRIPTION
**Pull request by dennmuel**
Added media queries and css modifications to change the layout of the user interface on smaller screened devices (e.g. smartphones and tablets). Also some minor changes in the German translation file have been made and would need to be adjusted in other files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,5 @@
 /palma.db
 /palma.ini
 /uploads
-/palma.po
```
Comment (stweil) [inline .gitignore]: Why did you change `.gitignore`?

Comment (dennmuel) [inline .gitignore]: This was a debug change I made, when I could not get my modified language file to be pulled and applied on my test machine. I probably forgot to undo it. I did so now.

File: font-awesome/css/font-awesome.css [hunk 2]
```diff
@@ -1,1338 +1,4 @@
 /*!
  *  Font Awesome 4.0.3 by @davegandy - http://fontawesome.io - @fontawesome
```
Comment (stweil) [inline font-awesome/css/font-awesome.css]: Did you use some tool to compress the original code? That makes it difficult to see what changed. Also updates to newer versions of Font Awesome might be made impossible.

Comment (dennmuel) [inline font-awesome/css/font-awesome.css]: I cannot recall making any change to this file.

File: index.php [hunk 3]
```diff
@@ -1,1249 +1,1300 @@
...
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>PalMA</title>
+
+<link rel="icon" href="theme/<?=CONFIG_THEME?>/favicon.ico" type="image/x-icon">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
+<link rel="stylesheet" href="pure-min.css">
+<link rel="stylesheet" href="palma.css" type="text/css">
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (stweil) [inline index.php]: This new line adds a dependency on jquery. Do we need it? If yes, we should use a local copy of jquery.min.js (which is already available on the PalMA stations) and update the documentation.

File: index.php [hunk 4]
```diff
@@ -1,1249 +1,1300 @@
...
+}
+
+function miniDisplaySelect(element) {
+    sendToNuc('layout=' + element.id);
+}
+
+function getHandlerCommand(handle, task) {
+
+    // console.log("getHandlerCommand "+handle+" - "+task);
+    // to deactivate buttons
```
Comment (stweil) [inline index.php]: Wrong indentation?

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 5]
```diff
@@ -213,11 +225,11 @@ msgstr "Linux-Benutzer können ihr X Display wie folgt teilen:"
 
 #: index.php:1218
 msgid "Share desktop (free UltraVNC / Windows only)"
-msgstr "Desktopfreigabe (UltraVNC (freie Software) / Windows)"
+msgstr "Desktopfreigabe (Windows)"
 
 #: index.php:1225
 msgid "Share desktop (non free RealVNC)"
-msgstr "Desktopfreigabe (RealVNC (kommerziell) / OS X)"
+msgstr "Desktopfreigabe (Mac)"
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: Here we should use the official OS‌ name `macOS`. What about all other occurrences of RealVNC?

File: login.php [hunk 6]
```diff
@@ -90,18 +90,18 @@ function checkCredentials($username, $password) {
 
     if (!$out) {
         trace("curl failed for user '$username'");
-        $errtext = __('Invalid credentials!');
+        $errtext = _('Invalid credentials!');
```
Comment (stweil) [inline login.php]: Is this modification here and also below intentional?

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po [hunk 7]
```diff
@@ -131,6 +131,18 @@ msgstr "Hilfe + Extras"
 msgid "Team display"
 msgstr "Team-Monitor"
 
+#: index.php:1090
```
Comment (stweil) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: These additions look strange because there are no corresponding additions for the other languages.

File: palma.css [hunk 8]
```diff
@@ -1,506 +1,781 @@
...
+/* Login form (login.php) */
+
+legend {
+    font-family: sans-serif;
+    font-size: 24px;
+    text-align: center;
+}
+
+#login_mask_outer {
+	padding: 1% 5% 1% 5%;
```
Comment (stweil) [inline palma.css]: Inconsistent indentation here

File: palma.css [hunk 9]
```diff
@@ -1,506 +1,781 @@
...
+	min-width: 220px;
+}
+
+#login_mask {
+	margin: 1% auto;
```
Comment (stweil) [inline palma.css]: Inconsistent indentation here and below

File: pure-min.css [hunk 10]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (stweil) [inline pure-min.css]: This is definitely wrong.

Comment (dennmuel) [inline pure-min.css]: Git sometimes does that to files. I deleted these lines whenever I saw them. That one slipped through.

File: palma.css (hunk 9 above)
Comment (dennmuel) [inline palma.css]: Fixed.

File: palma.css (hunk 8 above)
Comment (dennmuel) [inline palma.css]: Fixed.

File: login.php (hunk 6 above)
Comment (dennmuel) [inline login.php]: No, these are not intentional. Could be a problematic setting in my texteditor.

File: index.php (hunk 4 above)
Comment (dennmuel) [inline index.php]: Sorry, I don't quite understand what you mean.

File: index.php (hunk 3 above)
Comment (dennmuel) [inline index.php]: Yes, jquery is used for toggeling displayed contents. I will change the link.

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po (hunk 5 above)
Comment (dennmuel) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: "macOS" would be fine. What do you think about removing the application name in the msgid and use operating system names instead?

File: locale/de_DE.UTF-8/LC_MESSAGES/palma.po (hunk 7 above)
Comment (dennmuel) [inline locale/de_DE.UTF-8/LC_MESSAGES/palma.po]: These additions were made in one language only for developing purposes. Up to now they were not added, because they had not "officially" been approved yet.
